utils/near-rate-limiter/src/framed_read.rs

```diff
@@ -254,6 +277,8 @@ where
 
             let bytect = ready!(poll_read_buf(pinned.inner.as_mut(), cx, &mut state.buffer)?);
 
+            pinned.throttle_controller.report_bandwidth_used(bytect);
+
```

Comment:
can you add a test that checks that throttler.compute_msg_seen / consume_bandwidth_used is working.